flow/designs/asap7/mock-array/config.mk

```diff
@@ -28,6 +28,13 @@ export GDS_ALLOW_EMPTY       = Element
 
 export MACRO_PLACEMENT_TCL   = ./designs/asap7/mock-array/macro-placement.tcl
 
+# halo / placement blockage / trench around macro can be as small as 0.5
+export MACRO_PLACE_HALO      = 0.5 0.5 
```

Comment:
move these to defaults.


some of these are not used for manual placement, check docs and delete as approperiate.